apps-rendering/src/components/Deadline/styles.ts

```diff
@@ -0,0 +1,15 @@
+import { css } from '@emotion/react';
+import { brandAlt, neutral, remSpace } from '@guardian/source-foundations';
+import { darkModeCss } from 'styles';
+
+export const highlight = css`
+	color: ${neutral[7]};
+	display: flex;
+	align-items: center;
+	padding: 0 ${remSpace[1]};
+	font-family: GuardianTextSans;
```

Comment:
we usually set a font using some [helpers from `source-foundations`](https://github.com/guardian/csnx/blob/main/libs/%40guardian/source-foundations/src/typography/index.ts#L70). I'm not certain on the desired font size here, but it'd usually look something like

```suggestion
	${textSans.xsmall()}
```

Reply:
Thanks! Thats a good point, some of this best practice stuff i've not still got to grips with. Will update this now!